words.html

```diff
@@ -1160,31 +1162,31 @@
 </p>
 
 <p id="label_step">
-step
+krok
 </p>
 
 <p id="label_reset">
-reset
+resetuj
 </p>
 
 <p id="label_population">
-population
+populacja
```

Comment:
!TODO!

Zmieniłem na „gracze”, w tym kontekście lepiej pasuje - to zakładka w Sandboksie (7. Pobaw się).